src/utils/staking.js

```diff
@@ -1 +1,23 @@
+import {
+  extractTime, bnMin, BN_MAX_INTEGER, BN,
+} from '@polkadot/util';
+
 export const eraToDays = (era) => Math.round(era.toNumber() / 4);
```

Comment:
`eraToDays` is unused now.

Reply:
`eraToDays` is used in `UnbondModal` to make bondingDuration a time in days